tool/tctl/common/plugins_command.go

```diff
@@ -96,11 +125,59 @@ func (p *PluginsCommand) Cleanup(ctx context.Context, clusterAPI *auth.Client) e
 	return nil
 }
 
+func (p *PluginsCommand) InstallSCIM(ctx context.Context, client *auth.Client) error {
+	scimTokenHash, err := bcrypt.GenerateFromPassword([]byte(p.install.scim.token), bcrypt.DefaultCost)
+	if err != nil {
+		return trace.Wrap(err)
+	}
+
+	request := &pluginsv1.CreatePluginRequest{
+		Plugin: &types.PluginV1{
+			SubKind: types.PluginSubkindAccess,
+			Metadata: types.Metadata{
+				Labels: map[string]string{
+					types.HostedPluginLabel: "true",
+				},
+				Name: p.install.name,
+			},
+			Spec: types.PluginSpecV1{
+				Settings: &types.PluginSpecV1_Scim{
+					Scim: &types.PluginSCIMSettings{
+						SsoConnectorId: p.install.scim.samlConnector,
```

Comment:
Can we validate okta connector  and check if it exists   by doing following single api call ? 

`GetSAMLConnector(ctx, p.install.scim.samlConnector, false`